src/views/product-integrations-landing/contexts/integrations-search-context.tsx

```diff
@@ -10,7 +10,7 @@ import {
 	Tier,
 	IntegrationType,
 } from 'lib/integrations-api-client/integration'
-import { createContext, useContext, useMemo } from 'react'
+import { type ReactNode, createContext, useContext, useMemo } from 'react'
```

Comment:
Only if you need it... there's a `PropsWithChildren<T>` type that accepts an optional type argument

Reply:
Ahh word, good to know! 🙇